types/capella/withdrawals.yaml

```diff
@@ -0,0 +1,7 @@
+Capella:
+  Withdrawals:
+      type: array
+      items:
+        allOf:
+          - $ref: '../withdrawal.yaml#/Withdrawal'
```

Comment:
should be able to be just a ref? not sure we need `allOf` if we're not overriding anything...

Reply:
Removed where applicable.